--- llvm-project-3.9.1/llvm/utils/TableGen/AsmWriterEmitter.cpp	2020-04-30 17:50:05.260655100 +0200
+++ llvm-project-3.9.1/llvm/utils/TableGen/AsmWriterEmitter.cpp	2020-04-30 17:52:14.731996600 +0200
@@ -727,7 +727,7 @@
 namespace {
 struct AliasPriorityComparator {
   typedef std::pair<CodeGenInstAlias, int> ValueType;
-  bool operator()(const ValueType &LHS, const ValueType &RHS) {
+  bool operator()(const ValueType &LHS, const ValueType &RHS) const {
     if (LHS.second ==  RHS.second) {
       // We don't actually care about the order, but for consistency it
       // shouldn't depend on pointer comparisons.
